tests/test_aws.py

```diff
@@ -62,3 +62,57 @@ def test_aws_scan(self):
             if u["name"] == "startupos-test-bucket":
                 v = u["size"]
         assert v >= 262183
+
+
+@patch('verinfast.user.__get_input__', return_value='y')
+def test_aws_dash(self):
+    sub_id = "436708548746"
+    assert verinfast.user.initial_prompt is not None
+    file_path = Path(__file__)
+    test_folder = file_path.parent.absolute()
+    results_dir = test_folder.joinpath("results").absolute()
+    cfg_path = test_folder.joinpath("aws_dash.yaml").absolute()
+    agent = Agent()
+    config = Config(cfg_path=str(cfg_path))
+    assert config.cfg_path == str(cfg_path)
+    assert config.config is not FileNotFoundError
+    assert "modules" in config.config
+    config.output_dir = str(results_dir)
+    config.runGit = False
+    agent.config = config
+    agent.debug = DebugLog(path=agent.config.output_dir, debug=False)
+    agent.log = agent.debug.log
+    assert agent.config.output_dir == str(results_dir)
+    assert agent.config.dry is False
+    assert agent.config.runGit is False
+    assert len(agent.config.modules.cloud) > 0
+    try:
+        shutil.rmtree(results_dir)
+    except Exception as e:
+        print(e)
+    assert Path(results_dir).exists() is False
+    os.makedirs(agent.config.output_dir, exist_ok=True)
+    agent.scan()
```

Comment:
How will the test environment have the aws cli and access to that account?

Reply:
https://github.com/StartupOS/verinfast/settings/secrets/actions

Reply:
But also we need to use the action too. https://github.com/StartupOS/verinfast/blob/main/.github/workflows/release.yml#L68-L73